### ISSUE TITLE
read ECONNRESET

### ISSUE THREAD

**Issue by Wyleee** [BeepIsla/csgo-commend-bot#411]
I'm a computer novice Please help me tks！

 Checking for new update...
Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:183:27) {
  errno: 'ECONNRESET',
  code: 'ECONNRESET',
  syscall: 'read'
} Promise {
  <rejected> Error: read ECONNRESET
      at TLSWrap.onStreamRead (internal/stream_base_commons.js:183:27) {
    errno: 'ECONNRESET',
    code: 'ECONNRESET',
    syscall: 'read'
  }
}

**Comment by BeepIsla** [comment]
Something is blocking connection to Github to check for updates, either disable update check (Not recommended) or fix whatever is blocking Github.

**Comment by Wyleee** [comment]
Thank you. I have solved this problem, but I have a new one.

Checking for new update...
 Up to date!
 Checking protobufs...
 Found protobufs
 Opening database...
 There are a total of 15 operational accounts
 Parsing target account...
TypeError: Cannot read property 'switchProxyEveryXaccounts' of undefined
    at C:\Users\wyle\Desktop\csgo-commend-bot-master\index.js:250:34 Promise
 {
  <rejected> TypeError: Cannot read property 'switchProxyEveryXaccounts' of unde
fined
      at C:\Users\wyle\Desktop\csgo-commend-bot-master\index.js:250:34
}

Thanks for your help,﻿ε≡٩(๑>₃<)۶
